src/features/trustedDevice/pages/SuspensionChoice.tsx

```diff
@@ -19,8 +22,26 @@ import { getSpacing, Spacer, Typo } from 'ui/theme'
 import { SPACE } from 'ui/theme/constants'
 
 export const SuspensionChoice = () => {
-  const navigation = useNavigation<UseNavigationType>()
-  const onPressContinue = () => navigation.navigate('SuspensionConfirmation')
+  const { params } = useRoute<UseRouteType<'SuspensionChoice'>>()
+  const { navigate } = useNavigation<UseNavigationType>()
+  const { showErrorSnackBar } = useSnackBarContext()
+
+  const { mutate: suspendAccountForSuspiciousLogin, isLoading } =
+    useSuspendForSuspiciousLoginMutation({
+      onSuccess: () => {
+        navigate('SuspensionConfirmation')
+      },
+      onError: (error) => {
+        eventMonitoring.captureException(error)
```

Comment:
Il faut préfixer cette erreur d'un message "générique" qui concerne la suspension de compte pour la retrouver facilement. Il faut aussi la faire remonter comme un message de type `info`
```suggestion
        eventMonitoring.captureMessage(
          `Can't suspend account for suspicious login ; reason: "${
            error instanceof Error ? error.message : undefined
          }"`,
          'info'
        )
```